### PR TITLE
T10684: Enable Special:Mute

### PR DESCRIPTION
**Pull request by brandon-wm**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -2475,6 +2475,15 @@
 	'wgAllowHTMLEmail' => [
 		'default' => true,
 	],
+	'wgEnableSpecialMute' => [
+		'default' => true,
+	],
+	'wgEnableUserEmailMuteList' => [
+		'default' => true,
+	],
+	'wgEchoPerUserBlacklist' => [
```
Comment (redbluegreenhat) [inline LocalSettings.php:2484]: This should be moved to the Echo section, at line 1230.

**Review by redbluegreenhat — CHANGES_REQUESTED** [review]
Just a minor thing.

File: LocalSettings.php (hunk 1 above)
Comment (brandon-wm) [reply LocalSettings.php]: Apologies, done

File: LocalSettings.php [hunk 2]
```diff
@@ -1251,6 +1251,9 @@
 	'wgEchoMaxMentionsInEditSummary' => [
 		'default' => 0,
 	],
+		'wgEchoPerUserBlacklist' => [
```
Comment (redbluegreenhat) [inline LocalSettings.php:1254]: Too much indentation, remove one level.

**Review by redbluegreenhat — CHANGES_REQUESTED** [review]

**Review by redbluegreenhat — APPROVED** [review]

**Pull request merged**